Marlin/Configuration.h

```diff
@@ -1551,8 +1588,24 @@
  *  - Change to green once print has finished
  *  - Turn off after the print has finished and the user has pushed a button
  */
-#if ENABLED(BLINKM) || ENABLED(RGB_LED) || ENABLED(RGBW_LED)
+#if ENABLED(BLINKM) || ENABLED(RGB_LED) || ENABLED(RGBW_LED) || ENABLED(LEDSTRIP)
   #define PRINTER_EVENT_LEDS
+  #define LED_reset_time 60      // Set time for LEDs to turn off. (secs)
+  //#define NO_PAUSE_OR_TIMEOUT  // Removes the pause for click or timeout on event 0 (End of print).
+  //#define LIGHT_ON_POWERUP      // Light the LED(s) WHITE on startup.
+#endif
+
+#if ENABLED(PRINTER_EVENT_LEDS)
+  /* Printer events color selection */
+  #define homing                  2  // Yellow
+  #define auto_leveling           7  // Blacklight
+  #define manual_leveling         1  // White
+  #define printing                1  // White
+  #define filamentchange          4  // Aqua
+  #define filamentchange_timeout  5  // Aqua Dimmed
+  #define filemantchange_heat     6  // Aqua Half
+  #define printing_done           0  // Green
+  #define all_off                 9  // Off
```

Comment:
- Should be defined in an `enum` block.
- Should be named with `ALL_UPPER_CASE`.
- Should be in `enum.h` instead of `Configuration.h`.